libs/tx-statemachine/src/transitions/toCoin.ts

```diff
@@ -13,7 +13,7 @@ type Converter = (v: Decimal, rate: FXRate) => Decimal;
 export function toCoin(container: AnyActionContainer) {
   return doConversion(container, "fiat", "coin", (val, rate) =>
     toCoinDecimal(val
-      .div(rate.fxRate * rate.buy)
+      .div(rate.fxRate * rate.sell) // the rate we sell at
```

Comment:
woops!  Turns out replaying txs was a -very- good idea..a

Reply:
That is quite a different line